### ISSUE TITLE
DB-GPT-Web后续如何在DB-GPT中使用

### ISSUE THREAD

**Issue by yysirs** [csunny/DB-GPT-Web#1]
1.  文档问题
ReadME中
https://github.com/csunny/DB-GPT-Web#use-in-db-gpt
使用```npm run dev compile```会报错，修改成```npm run compile```. 就成功了。

2. 后续使用问题
编译完成后，生成内容如下：
<img width="121" alt="fbfd60a459b553b29cfc524b1038210" src="https://github.com/csunny/DB-GPT-Web/assets/43271630/134e23aa-6489-43f7-8b24-9e5b14977ee5">

出现404.html，是否正常

3. 如何结合DB-GPT使用
按照README文档教程，并没有说明后续在DB-GPT中使用，是自动调用还是需要触发？这部分内容是否完善的能更加清楚一些。

**Comment by csunny** [comment]
> 1. 使用`npm run dev compile`会报错，修改成`npm run compile`. 就成功了。

Yes, `npm run dev compile` is a incorrect command,  use `npm run compile`.    if you already compile successfully, just copy file to DB-GPT path,  command as follows
```
cp -r -f /Db-GPT-Web/out/* /DB-GPT/pilot/server/static/*
```

**Comment by ykhorzon** [comment]
It should "cp -r -f /Db-GPT-Web/out/* /DB-GPT/pilot/server/static/" instead of "cp -r -f /Db-GPT-Web/out/* /DB-GPT/pilot/server/static/*". The extra * in destination will create sub-directory in /DB-GPT/pilot/server/static/.
